zerver/models.py

```diff
@@ -2219,11 +2220,11 @@ class Meta:
         abstract = True
 
 class ScheduledEmail(AbstractScheduledJob):
-    # Exactly one of user or address should be set. These are used to
+    # Exactly one of users or addresses should be set. These are used to
     # filter the set of ScheduledEmails.
-    user = models.ForeignKey(UserProfile, null=True, on_delete=CASCADE)  # type: Optional[UserProfile]
+    users = models.ManyToManyField(UserProfile)  # type: Manager
     # Just the address part of a full "name <address>" email address
-    address = models.EmailField(null=True, db_index=True)  # type: Optional[str]
+    addresses = ArrayField(models.EmailField(null=True, db_index=True), null=True)  # type: List[Optional[str]]
```

Comment:
https://docs.djangoproject.com/en/2.1/ref/contrib/postgres/fields/#indexing-these-fields suggests this may not be a good way to index this.

Reply:
It's possible we should just keep `address` as a singleton -- we don't actually have a use case for sending to multiple email addresses without corresponding users, yet.  Losing the database index is painful, and it's probably more work to replace the DB index...